unison-core/src/Unison/Util/Relation.hs

```diff
@@ -112,15 +112,25 @@ outerJoinRanMultimaps :: (Ord a, Ord b, Ord c)
                       -> Map c (Set a, Set b)
 outerJoinRanMultimaps a b = outerJoinDomMultimaps (swap a) (swap b)
 
-innerJoinDomMultimaps :: (Ord a, Ord b, Ord c)
```

Comment:
Oops, I branched off of another PR that hasn't merged yet. Will remove this from the diff after that lands.